File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    
    do 
    {
        printf("Height: ");
        n = get_int();
    }
    while ( n < 0 || n > 23); 
    
   for (int i= n; i > 0 ; i--) {
       
        for (int s= 2; s <= i; s++)
       
      printf(" ");
       
       for (int p=n + 1; p >= i; p--)
       
      printf("#");
      printf("\n");

       }

   
}

File: pset6/mario.py
import cs50

height = int(input("Height: "))

if height > 0 and height < 23:
    
    x = " "
    spaces = height
    hashes = 2
    
    print(hashes)

    for i in range(0,height):
        for s in range(1,spaces):
            print(x,end="")
            
        for h in range(0,hashes):
            print("#",end="")
            
        spaces -= 1 
        hashes += 1
        print()
        
            
        
        
